.github/workflows/config.yml

```diff
@@ -37,7 +36,6 @@ jobs:
     - name: Run tests
       run: |
         python -m pip install coverage
```

Comment:
We have `hypothesis` in `requirements.txt` (and specifically excluded from `setup.py`: https://github.com/Axelrod-Python/Axelrod/blob/master/setup.py#L4). Should we do the same with `coverage`, `mypy`, `isort`? (Just do be consistent.)

(Happy for this to be done in a later PR - just a thought.)

Reply:
Sounds good. It would be good to also have a "presubmit" script to run isort, mypy, etc. locally in line with config.yml.

Reply:
Good idea.

Reply:
Opened #1360